.env.example

```diff
@@ -0,0 +1,4 @@
+AUTOTRAIN_USERNAME=autoevaluator # The bot that authors evaluation jobs
```

Comment:
I'm not sure why these changes are appearing in your branch - maybe you need to rebase / force push?

Reply:
oop sorry ok I just checked — it's because the feature branch I'm merging into isn't up-to-date. Can you merge main on the feat/add_zero_shot_classification branch? They should go away then! 

I've also added a small change which renames the task to text_zero_shot_classification as requested by the autotrain team.

Reply:
Thanks, I've just brought `feat/add_zero_shot_classification` up to date with `main` - you'll now need to bring your current branch up to date with `feat/add_zero_shot_classification` to eliminate these finals diffs (I think!)